### PR TITLE
August 14 Notes

### PR DESCRIPTION
**Pull request by AGS-**
Putting this out for feedback on the notes format.

### CODE REVIEW

File: notes/20170814_meeting.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# GraphQL WG Meeting #1 Notes (August 14, 2017)
```
Comment (leebyron) [inline notes/20170814_meeting.md]: Could we rename this file to `2017-08-14.md` to match the naming scheme in https://github.com/graphql/graphql-wg/tree/master/agendas

File: notes/20170814_meeting.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Martijn Walraven | Apollo | Amsterdam, The Netherlands
+Ian MacLeod   | Convoy        | Seattle, WA
+Robert Mosolgo   | graphql-ruby | Charlottesville, VA
+Thea Lamkin   | Apollo        | San Fancisco, CA 
+
+## Agenda
+
+* Discuss upcoming meeting schedule
+	* Johannes mentioned that it would be nice to have a time shift for the next meeting because he's going to Australia. 
+* [GCL: GraphQL Configuration Language](https://github.com/graphcool/gcl)
```
Comment (leebyron) [inline notes/20170814_meeting.md]: How about a ### for each agenda item, and then bullets below it?

**Comment by AGS-** [comment]
Cool sounds good, I'll fix this and finish off the rest of the agenda points today.

**Comment by AGS-** [comment]
Updated

**Comment by wincent** [comment]
Thanks for doing all this work, @AGS-. Next time let's make sure we share the load around so that you don't have to take notes for 3 hours!

**Pull request merged**